src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/CNNScoreVariants.java

```diff
@@ -221,7 +219,7 @@
     private int windowStart = windowSize / 2;
     private boolean waitforBatchCompletion = false;
 
-    private File scoreFile;
+    private File scoreFile; // use java.nio.File here because python code needs to write to this
```

Comment:
This is `java.io.File`, not `java.nio.File`

Reply:
Oh right - fixed.